README.md

```diff
@@ -1 +1,178 @@
-# motorbike_booking_backend
\ No newline at end of file
+# Motorbike Booking Back-End
+
+> Motorbike Booking is a rails backend application for booking Motorbikes. This application serves as API endpoints for the frontend and users can load, create, and delete Motorbikes, as well as add reservations. Users need to sign up or sign in to access the API endponits. They can sign up or sign in using the API endponts.
+
+## Schema
+![](./app/assets/images/ERD.png)
+
+### Front-End Repository
+
+[Motorbike Booking Front-end](https://github.com/RWUBAKWANAYO/motorbike_booking_frontend)
+
+### Kanban Board
+[Kanban Board](https://github.com/RWUBAKWANAYO/motorbike_booking_backend/projects/1)
+
+### Intial State of Kanban Board
+[Initial Kanban Board](./app/assets/images/Kanban.png)
+
+> ***We are a group of five***
+1. Member 1 - [Pedro](https://github.com/ppmarq1)
+2. Member 2 - [Joseph](https://github.com/Kwentsir)
+3. Member 3 - [Emmanuel](https://github.com/Epaltechs)
+4. Member 4 - [Ghilain](https://github.com/Ghilain)
+5. Member 5 - [Olivier](https://github.com/RWUBAKWANAYO)
+
+## Built With
+
+- Major languages (Ruby)
+- Framworks (Ruby on Rails)
+- Testing library (Rspec)
+- API docs (Rswag)
+
+## Live version
+
+[In progress]
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A text editor(preferably Visual Studio Code)
+
+### Install
+- Ruby
+- Ruby on Rails
+- PostgresSQL
+
+### Using it Locally
+
+- Clone the project
+```
+git clone https://github.com/RWUBAKWANAYO/motorbike_booking_backend
+cd motorbike_booking_backend
+```
+
+### Setup
+
+Add your username & password in `database.yml` file:
+```
+username: <your_username>
+password: <your_password>
+```
+Specify your ruby version if different from mine
+```
+ruby 'Your_ruby_version'
+```
+
+Install gems with:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+rails db:seed
+```
+### Usage
+
+Start server with:
+
+```
+rails server -p 3000
+```
+
+Visit http://localhost:3000/ in your browser.
+
+### Run tests
```

Comment:
- [ ] I can run the tests now, however 20 are failing:

![image](https://user-images.githubusercontent.com/57421823/195943698-cf1b5a5c-0f9d-4be0-be8e-4bf6e42d1362.png)

It seems the problematic tests are the ones that expect a response (e.g.  200)

Please, debug those tests to ensure the integrity of your app.